packages/co-busboy/src/index.ts

```
@@ -0,0 +1,373 @@
+
+  return parts;
+
+  function onField(name: string, val: string, info: FieldInfo): void {
+    const fieldnameTruncated = info.nameTruncated;
+    const valTruncated = info.valueTruncated;
+
+    if (checkField) {
+      const err = checkField(name, val, fieldnameTruncated, valTruncated);
+      if (err) {
+        debug('onField error: %s', err);
+        return onError(err);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)